src/components/Identity/index.js

```diff
@@ -100,6 +146,12 @@ const createIdentity = ({ config, logger, cookie }) => {
     commands: {
       getEcid() {
         return optIn.whenOptedIn().then(getEcid);
+      },
+      setCustomerIDs(options) {
```

Comment:
Should this be `setCustomerIds` ?

Reply:
I just used the existing API name in Visitor JS today.